team/brent-wilder.yaml

```diff
@@ -0,0 +1,19 @@
+- title: Brent Wilder
+  avatar: https://avatars.githubusercontent.com/brentwilder
+  role: PhD Student
+  organizations:
+    - name: Boise State University
+  bio: >
+    Brent is a PhD Student at Boise State University with a focus on modeling snow properties from imaging spectroscopy data. 
```

Comment:
Student? :smiley:

Reply:
@jomey life long learner ;) , lol but do not know what my title is really post defense and right before submitting dissertation. this works fine!